Automattic/jetpack
# Cover Block: Remove the upgrade banner

Fixes https://github.com/Automattic/view-design/issues/210

#### Changes proposed in this Pull Request:

* Remove the upgrade banner notice for video backgrounds. This is confusing users because it makes the whole cover block seem as though it is a premium block.


| Before      | After |
| ----------- | ----------- |
| <img width="1277" alt="106206346-c1c5d300-6174-11eb-9de2-50b16004c872" src="https://user-images.githubusercontent.com/1464705/106208154-ca6bd880-6177-11eb-971b-31209593ec1c.png"> | <img width="1353" alt="Screen Shot 2021-01-28 at 2 47 32 PM" src="https://user-images.githubusercontent.com/1464705/106208161-cd66c900-6177-11eb-8542-76b21ac6c609.png"> |

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






Unfortunately this change won't work with the Jetpack sync to sandbox, so you will need to manually remove: `projects/plugins/jetpack/extensions/extended-blocks/core-cover/core-cover.php` from your sandbox and reload the page.

Confirm that you do not see the upgrade banner when the block is selected.

#### Proposed changelog entry for your changes:


* None needed.
